docs/get-started/getting-started-alluxio.md

```diff
@@ -365,3 +365,77 @@ This section will give some links about how to configure, tune Alluxio and some
 - [Running Spark on Alluxio](https://docs.alluxio.io/os/user/stable/en/compute/Spark.html)
 - [Performance Tuning](https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html )
 - [Alluxio troubleshooting](https://docs.alluxio.io/os/user/stable/en/operation/Troubleshooting.html)
+
+## Alluxio reliability
+The properties mentioned in this section can be found in [Alluxio configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+### Auto start Alluxio masters and workers
+After installed Alluxio masters and workers, use this [script](../../scripts/alluxio/alluxio-systemd.sh) to create systemd services and start masters and workers.
+#### Prerequisites
+- Format Alluxio on masters: 
+    ```bash
+    $ ./bin/alluxio format
+    ```
+    Or you have executed `alluxio-start.sh master` which internally invoke `alluxio format`  
+- Close all the Alluxio masters and workers
```

Comment:
what does this mean?  stop all of the masters and workers?

Reply:
Will move the systemd script to the docker image. 
Wil test this when creating the docker image